config/plugins/visualizations/crystvis/config/crystvis.xml

```diff
@@ -7,33 +7,266 @@
+                    <data>
+                        <value>bonded</value>
+                        <label>Bonded</label>
+                    </data>
+                    <data>
+                        <value>molecule</value>
+                        <label>Molecule</label>
+                    </data>
+                    <data>
+                        <value>custom</value>
+                        <label>Custom</label>
+                    </data>
                 </data>
-            </data>
+            </test_param>
+            <cases>
+                <cases>
+                    <value>all</value>
+                </cases>
+                <cases>
+                    <value>indices</value>
+                    <inputs>
+                        <inputs>
+                            <label>Indices</label>
+                            <name>indices</name>
+                            <type>text</type>
+                            <value>0</value>
+                            <help>One or more space separated atom indices to show</help>
+                        </inputs>
+                    </inputs>
+                </cases>
+                <cases>
+                    <value>elements</value>
+                    <inputs>
+                        <inputs>
+                            <label>Elements</label>
+                            <name>elements</name>
+                            <type>text</type>
+                            <value>H</value>
+                            <help>One or more space separated chemical symbols to show</help>
+                        </inputs>
+                    </inputs>
+                </cases>
+                <cases>
+                    <value>cell</value>
+                    <inputs>
+                        <inputs>
+                            <label>Cell</label>
+                            <name>cell</name>
+                            <type>text</type>
+                            <value>0 0 0</value>
+                            <help>Show all atoms within the unit cell with the provided indices, i j k indices should be space separated</help>
+                        </inputs>
+                    </inputs>
+                </cases>
+                <cases>
+                    <value>box</value>
+                    <inputs>
+                        <inputs>
+                            <label>Box Start</label>
+                            <name>box_start</name>
+                            <type>text</type>
+                            <value>0 0 0</value>
+                        </inputs>
+                        <inputs>
+                            <label>Box End</label>
+                            <name>box_end</name>
+                            <type>text</type>
+                            <value>1 1 1</value>
+                            <help>All atoms with a co-ordinates between the start and end values will be returned, co-ordinates should be space separated</help>
```

Comment:
Minor typo - could do without the 'a'. Also perhaps 'returned' should be 'displayed' in this context?